### DISCUSSION TITLE
How to upgrade client version

### DISCUSSION THREAD

**Discussion by AlexLong152** [abraunegg/onedrive#2007] [Q&A] [answered]
Newbie question here, running the command ```onedrive --synchronize```   syncs with the cloud, but also prints:

```WARNING: Your onedrive client version is obsolete and unsupported. Please upgrade your client version.```

What is the best way to upgrade the client version? I couldn't find it in the docs.

My version is: ```onedrive v2.4.18-16-g83a7907```

I'm using Pop!_OS 22.04

**Comment by abraunegg** [comment, marked as answer]
@AlexLong152 
> What is the best way to upgrade the client version? I couldn't find it in the docs.

I guessed you missed reading this: https://github.com/abraunegg/onedrive/blob/master/docs/INSTALL.md#upgrading-the-client

> I'm using Pop!_OS 22.04

As you are using a distribution based on Ubuntu, you are better off [uninstalling your manually installed client](https://github.com/abraunegg/onedrive/blob/master/docs/INSTALL.md#uninstalling-the-client) then configuring your system to use the OpenSuSE Build Service Packages as per https://github.com/abraunegg/onedrive/blob/master/docs/ubuntu-package-install.md

This means your client can be upgraded automatically when the packages are updated.
